samples/excel/85-preview-apis/worksheet-find-all.yaml

```diff
@@ -0,0 +1,154 @@
+order: 4
+id: excel-worksheet-find-all
+name: Find all ranges
```

Comment:
Even though this name describes what the API is doing (it's finding ranges), it doesn't sound like what the snippet does, from a functional perspective. Snippet names should describe what the snippet does from a functional perspective, because that's how the user is likely to think of things (i.e., a user is more likely to think in terms of "I want to find data in a worksheet" than "I want to find all ranges"). 

And, I just noticed that the previous snippet (`excel-range-find`) is named **Find** -- probably should rename that snippet as well.

- Consider renaming this snippet to "Find data in worksheet"
- Consider renaming the `excel-range-find` snippet to "Find data in range"